nifi-nar-bundles/nifi-standard-services/nifi-mongodb-services-bundle/nifi-mongodb-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,131 @@
+nifi-lookup-services-nar
```

Comment:
Does this file need modified for this nar?

Reply:
Not sure. Does it matter?